index.html

```diff
@@ -13,53 +13,162 @@
+            </ul>
+        </div>
+    </section>
+
+    <section class="all1">
+        <img class="one" src="images/SnapshootPortfolio(2).jpg" alt="">
+        <h2 class="tonic">Tonic</h2>
+        <div class="list">
+            <ul class="inline">
+                <li><img class="words" src="images/Frame2.jpg" alt=""></li>
+            </ul>
+        </div>
+        <h3 class="subtitle">A daily selection is privaly 
+            personalized reads; 
+            no accounts or sign-ups required
+        </h3>
+        <div class="leng">
+            <ul>
+                <li><img class="lenguage" src="images/Tags.jpg" alt=""></li>
+                <li><a href="https://github.com/luccamicale"><img class="see" src="images/Enabled.jpg" alt=""></a></li>
+            </ul>
+        </div>
+    </section>
+
+    <section class="all2">
+        <img class="two" src="images/SnapshootPortfolio)1.jpg" alt="">
+        <h2 class="multipost">Multi - Post 
+            Stories
+        </h2>
+        <div class="list2">
+            <ul class="inline2">
+                <li><img class="words2" src="images/Frame2.jpg" alt=""></li>
+            </ul>
+        </div>
+        <h3 class="subtitle1">A daily selection is privaly 
+            personalized reads; 
+            no accounts or sign-ups required
+        </h3>
+        <div class="leng">
+            <ul>
+                <li><img class="lenguage" src="images/Tags.jpg" alt=""></li>
+                <li><a href="https://github.com/luccamicale"><img class="see" src="images/Enabled.jpg" alt=""></a></li>
+            </ul>
+        </div>
+    </section>
+
+    <section class="all3">
+        <img class="two" src="images/SnapshootPortfolio77.jpg" alt="">
+        <h2 class="multipost">Tonic
+        </h2>
+        <div class="list2">
+            <ul class="inline2">
+                <li><img class="words3" src="images/Frame2.jpg" alt=""></li>
+            </ul>
+        </div>
+        <h3 class="subtitle1">A daily selection is privaly 
+            personalized reads; 
+            no accounts or sign-ups required
+        </h3>
+        <div class="leng">
+            <ul>
+                <li><img class="lenguage" src="images/Tags.jpg" alt=""></li>
+                <li><a href="https://github.com/luccamicale"><img class="see" src="images/Enabled.jpg" alt=""></a></li>
+            </ul>
+        </div>
+    </section>
+
+    <section class="all4">
+        <img class="two" src="images/SnapshootPortfolio)1.jpg" alt="">
+        <h2 class="multipost"> Multi - Post
+            Stories
+        </h2>
+        <div class="list2">
+            <ul class="inline2">
+                <li><img class="words2" src="images/Frame2.jpg" alt=""></li>
+            </ul>
+        </div>
+        <h3 class="subtitle1">A daily selection is privaly 
+            personalized reads; 
+            no accounts or sign-ups required
+        </h3>
```

Comment:
Great design @luccamicale 🤝

- [ ] You may kindly use a `p` tag instead of an `h3` for this section.

Reply:
Good job following the suggestions of the previous reviewer